rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+All component repos that will assign items to a DCP Milestone MUST be added to the ZenHub *DCP Backlogs* workspace.
+
+DCP Product Backlog items  are modeled as:
+- [User Stories](http://agiledictionary.com/277/user-story/) with acceptance criteria
+- [Epics](http://agiledictionary.com/309/epic/) contain a related set of smaller User Stories required to complete an objective.
+- [Spikes](http://agiledictionary.com/209/spike/) adopted from eXtreme Programming to answer design questions.
+  - [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+#### User Stories
+
+The [*researcher with a keyboard*](https://docs.google.com/drawings/d/1Li9o_uVkEZDZHbkQv86I49FG8I0zGSgHYkKabp7piEs/edit) and [*researcher with a pipette*](https://docs.google.com/drawings/d/1o6I5BLtJuRJ1vUdb7dHCkFbo0XiaxKDBiVAogqnUowU/edit) journeys are the initial source of user stories that  are modeled as epics in the *dcp* repo,  reflecting the narrative flow of the product. Further refinement by the Product Owners is required to both develop and  extend the high-level set of user activities. 
+
+Since journeys may vary per persona, personas MUST be identified for an epic by assigning a documented GitHub label such as "keyboard", "pipette", or "road-builder".  An epic MAY support multiple personas.
+
+#### Technical and Operational Requirements
+Product-wide technical or operational requirements are defined by **Technical Architecture** and modeled as epics in the *dcp* repo.
+
+#### Owners
+
+A DCP owner MUST be assigned to each issue or epic in the *Product Backlog* in the *dcp* repo to shepherd and coordinate progress. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to epics in the *dcp* repo.
+
+For example, an implementation task for a specific component resides in its own ZenHub repo and is linked to the relevant epic in the *dcp* repo.
+
+* No component-specific GitHub labels such as "Data Store" MUST be assigned to an epic in the *dcp* repo. The link documents the relationship.
+
+### Sprints
+
+ **Four week sprints**. There MUST be one DCP Sprint and Sprint Backlog, not a separate Sprint and Sprint Backlog for each component team. Each DCP Sprint starts on Thursday and ends on Wednesday.  
+ 
+Each Sprint MUST be assigned a ZenHub *Milestone* with a due date by the **Product Owners**.
+
+ All Scrum events are scheduled on the DCP calendar at a regular cadence. For example:
+![](../images/0000-scrum-calendar.png)
+
+### Refinement
+
+New items appear in the ZenHub *New* pipeline and must be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline.
+
+**NOTE**: *To avoid maintenance overhead due to duplicate or incomplete issues, the creation of new items SHOULD be limited to **DCP PM** or a facilitator from **Technical Architecture**.*
```

Comment:
```suggestion
**NOTE**: *To avoid maintenance overhead due to duplicate or incomplete issues, the creation of new items SHOULD be limited to **DCP PM** or a facilitator from the Architecture Team.
```

Reply:
Also: this and the previous line probably belong in the `DCP Product Backlog` section.